### PR TITLE
PUB 1269 send media applications report

### PR DESCRIPTION
**Pull request by joshblackmoor**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1269

### Change description ###

- Added new endpoint to handle sending media requests
- Added new Csv creation service
- Added new env variable for the team email which comes from the key vault
- Converted maps to be <String, Object> rather than <String, String> (this is required for sending files and does not break existing notifications)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by joshblackmoor** [comment]
> Details

Done :)

**Comment by joshblackmoor** [comment]
> Looks good! Needs some integration tests adding for new the endpoint too

Done :)

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/CsvCreationService.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     */
+    public byte[] createMediaApplicationReportingCsv(List<MediaApplication> mediaApplicationList) {
+        try (StringWriter sw = new StringWriter(); CSVWriter csvWriter = new CSVWriter(sw)) {
+            csvWriter.writeNext(HEADINGS);
+            mediaApplicationList.forEach(application ->
+                csvWriter.writeNext(application.toCsvStringArray()));
+
+            return sw.toString().getBytes("UTF-8");
+        } catch (IOException e) {
+            log.error("Error occurred %s when trying to create a csv with list %s",
```
Comment (kurt-uk) [inline src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/CsvCreationService.java:37]: logging an error where youre throwing an exception isnt needed, the exception should handle the logging of whats gone wrong, especially if its our exception

**Review by kurt-uk — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/CsvCreationService.java (hunk 1 above)
Comment (joshblackmoor) [reply src/main/java/uk/gov/hmcts/reform/pip/publication/services/service/CsvCreationService.java]: Ah yes. Removed that line 👍

**Review by kurt-uk — APPROVED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**